content/docs/setup/kubernetes/multicluster-install.md

```diff
@@ -43,35 +43,89 @@ Istio control plane, Envoy can then communicate with the **single**
+
+The following procedure should be performed on each remote cluster to be
+added to the service mesh.  The procedure requires cluster-admin user access
+to the remote cluster.
+
+1. Create a `ClusterRole` named `istio-reader` for the Istio control plane access:
+
+     ```command
+      cat <<EOF | kubectl  create -f -
+      kind: ClusterRole
+      apiVersion: rbac.authorization.k8s.io/v1
+      metadata:
+        name: istio-reader
+      rules:
+         - apiGroups: ['']
+           resources: ['nodes', 'pods', 'services', 'endpoints']
+           verbs: ['get', 'watch', 'list']
+    EOF
+    ```
 
-After deployment of remote clusters, each one will have a
-credentials file associated with the admin context typically
-located in `$HOME/.kube/config`.
+1.  Create a `ServiceAccount` named `istio-multi` for the Istio control plane:
 
-## Gather credential files from remote
+    ```command
+    $ export SERVICE_ACCOUNT=istio-multi
+    $ export NAMESPACE=istio-system
+    $ kubectl create ns ${NAMESPACE}
+    $ kubectl create sa ${SERVICE_ACCOUNT} -n ${NAMESPACE}
+    ```
 
-> `${CLUSTER_NAME}` here is defined as the name of the remote
-cluster used and must be unique across the mesh.  Some Kubernetes
-installers do not set this value uniquely.  In this case, manual
-modification of the `${CLUSTER_NAME}` fields must be done.
+1.  Bind the `ServiceAccount` named `istio-multi` to the `ClusterRole` named `istio-reader`:
 
-For each remote cluster, execute the following steps:
+    ```command
+    $ kubectl create clusterrolebinding istio-multi --clusterrole=istio-reader --serviceaccount=${NAMESPACE}:${SERVICE_ACCOUNT}
+    ```
 
-1. Determine a name for the remote cluster that is unique across
-all clusters in the mesh.  Substitute the chosen name for the
-remaining steps in `${CLUSTER_NAME}`.
+1.  Prepare environment variables for building the `kubeconfig` file for `ServiceAccount` `istio-multi`:
 
-1. Copy the credentials file form the remote Kubernetes cluster
-to the local Istio control plane cluster directory
-`$HOME/multicluster/${CLUSTER_NAME}`.  The `${CLUSTER_NAME}` must
-be unique per remote.
+    ```command
+    $ export WORK_DIR=$(pwd)
+    $ CLUSTER_NAME=$(kubectl config view --minify=true -o "jsonpath={.clusters[].name}")
+    $ export KUBECFG_FILE=${WORK_DIR}/${CLUSTER_NAME}
+    $ SERVER=$(kubectl config view --minify=true -o "jsonpath={.clusters[].cluster.server}")
+    $ SECRET_NAME=$(kubectl get sa ${SERVICE_ACCOUNT} -n ${NAMESPACE} -o jsonpath='{.secrets[].name}')
+    $ CA_DATA=$(kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o "jsonpath={.data['ca\.crt']}")
+    $ TOKEN=$(kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o "jsonpath={.data['token']}" | base64 --decode)
+    ```
 
-1. Modify the name of the remote cluster's credential file field
-`clusters.cluster.name` to match `${CLUSTER_NAME}`.
+    __NOTE__: An alternative to `base64 --decode` is `openssl enc -d -base64 -A` on many systems.
```

Comment:
Do we need to mention `openssl enc -d -base64 -A`? I found that most of the Kubernetes command are using `base64 --decode`

Reply:
IMO, leaving it as a note doesn't hurt.